xDripG5/Messages/SessionStartTxMessage.swift

```diff
@@ -12,11 +12,13 @@ import Foundation
 struct SessionStartTxMessage: RespondableMessage {
     typealias Response = SessionStartRxMessage
 
-    let startTime: UInt32
+    let time: UInt32
```

Comment:
time and timeEpoch need more disambiguation, either via naming or documentation comments